sendgrid/helpers/stats/stats.py

```diff
@@ -110,21 +110,21 @@ def get(self):
         :return: response stats dict
         """
         stats = {}
-        if self.start_date is not None:
+        if self.start_date:
             stats["start_date"] = self.start_date
-        if self.end_date is not None:
+        if self.end_date:
             stats["end_date"] = self.end_date
-        if self.aggregated_by is not None:
+        if self.aggregated_by:
             stats["aggregated_by"] = self.aggregated_by
-        if self.sort_by_metric is not None:
+        if self.sort_by_metric:
             stats["sort_by_metric"] = self.sort_by_metric
-        if self.sort_by_direction is not None:
+        if self.sort_by_direction:
             stats["sort_by_direction"] = self.sort_by_direction
-        if self.limit is not None:
+        if self.limit:
```

Comment:
This is just a pure guess but what if `self.limit == 0`? This update wouldn't allow that to be properly set.